scripts/context.js

```diff
@@ -94,6 +96,10 @@ function singlePageView() {
           }
         }
       }
+      //Hide error message if a context is selected
+      if (countFeature > 0) {
```

Comment:
Please hide it by default, and show it if the count is <= 0 . Thank you

Reply:
I would like to learn the difference between these two cases. Please explain. Thanks :)

Reply:
Whenever i reload the context screen i could see the not context selected popup for a sec. This will solve that issue